cmd/api/database.go

```diff
@@ -42,6 +44,7 @@ func migrationUp() error {
 	if err != nil {
 		return fmt.Errorf("migrating up: %w", err)
 	}
+	mGlobal = m
 
 	m.Up()
```

Comment:
Not in this PR scope, but I think we forgot to get the error back from the function here.
```suggestion
err = m.Up()